NEWgIT.Client/Pages/Index.razor

```diff
@@ -0,0 +1,164 @@
+ICollection<string>? Forks;
+Dictionary<DateOnly, int>? frequencyDictionary;
+Dictionary<string, Dictionary<DateOnly, int>>? authorDictionary;
+Dictionary<string, commitItem[]> commitsPrAuthor = new Dictionary<string, commitItem[]>();
+string[] commitsPrAuthors;
+string? value = "";
+string? repoIdentifier; 
+commitItem[] allCommits;
+int stupidFixer = 0; 
+
+void OnChange(string value)
+{
+    this.value = value;
+}
+static readonly HttpClient client = new HttpClient();
+
+async void OnClick(string buttonName)
+{   
+    analyseAllCommits();
+    analysePrAuthor();
+    analyseForks();
+} 
+
+async void analyseAllCommits(){
+    try
+    {
+        HttpResponseMessage response = await client.GetAsync($"https://localhost:7235/analysis/{value}/frequency");
+        response.EnsureSuccessStatusCode();
+
+        var responseBody = await response.Content.ReadFromJsonAsync<FrequenciesDTO>();
+        var frequencyDictionary = responseBody.Frequencies;
+        allCommits = new commitItem[frequencyDictionary.Count];
+        convert(frequencyDictionary, allCommits);
+    }
+    catch (HttpRequestException e)
+    {
+        Console.WriteLine("\nException Caught!");
+        Console.WriteLine("Message :{0} ", e.Message);
+    }
+}
+
+async void analysePrAuthor(){
+     try
+    {
+        HttpResponseMessage responsePrAuthor = await client.GetAsync($"https://localhost:7235/analysis/{value}/author");
+        responsePrAuthor.EnsureSuccessStatusCode();
+
+        var responseBody = await responsePrAuthor.Content.ReadFromJsonAsync<AuthorsDTO>();
+        var authorDictionary = responseBody.AuthorFrequencies;
+        
+        foreach (var author in authorDictionary)
+        {  
+        commitItem[] authorArr = new commitItem[author.Value.Count];
+        convert(author.Value, authorArr);
+        commitsPrAuthor.Add(author.Key, authorArr);
+        } 
+    }
+    catch (HttpRequestException e)
+    {
+        Console.WriteLine("\nException Caught!");
+        Console.WriteLine("Message :{0} ", e.Message);
+    }
+}
+
+async void analyseForks(){
+    try
+    {
+        HttpResponseMessage response = await client.GetAsync($"https://localhost:7235/analysis/{value}/forks");
+        response.EnsureSuccessStatusCode();
+        var responseBody = await response.Content.ReadFromJsonAsync<ForksDTO>();
+        var Forks = responseBody.Forks;
+    }
+    catch (HttpRequestException e)
+    {
+        Console.WriteLine("\nException Caught!");
+        Console.WriteLine("Message :{0} ", e.Message);
+    }
+}
+
+void convert(Dictionary<DateOnly, int> toBeConverted, commitItem[] arr){
+    var sortedDictionary = new SortedDictionary<DateOnly, int>(toBeConverted);
+    int counter = 0; 
+    foreach (var date in sortedDictionary)
+    {
+        arr[counter] = new commitItem
+        {
+            Date = date.Key,
+            Commits = date.Value
+        };
+        counter++;        
+    }
+    StateHasChanged();
+}
+
+//Helper 
+class commitItem
+{

```

Comment:
Ser ud som om der er noget galt med formatet her?

Reply:
For at følge op på dette, alt der er inden for @code{ } skal lige indenteres en enkelt tab ind :)

Reply:
For at følge op på dette, ligner det at alt inden for @code{ ... } skal indenteres ind med et enkelt tab